lib/gis_robot_suite/structural_metadata_builder.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module GisRobotSuite
+  class StructuralMetadataBuilder
+    def self.build(*args)
+      new(*args).build
+    end
+
+    # @param [String] druid
+    # @param [Hash<Symbol,Assembly::ObjectFile>] objects
```

Comment:
What is the Symbol?

Reply:
Is it an array of ObjectFiles?

Reply:
🤷 not sure, will try to figure out...

Reply:
working backwards to the calling code in `generate_content_metadata.rb`: this param is a hash, where the keys are categories of file, and the values are indeed arrays of `Assembly::ObjectFile`.  updated the param name to `object_files` per a different comment, and made this YARD comment more specific and correct (hopefully).